src/WebJobs.Script.WebHost/Security/Authentication/AuthenticationLevelHandler.cs

```diff
@@ -50,6 +49,11 @@ internal class AuthenticationLevelHandler : AuthenticationHandler<Authentication
                     new Claim(SecurityConstants.AuthLevelClaimType, requestAuthorizationLevel.ToString())
                 };
 
+                if (!string.IsNullOrEmpty(name))
```

Comment:
Nice. I see some of the stuff I had to do to get EasyAuth integrated you're already handling now (identity, claims, etc.)

Reply:
Yeah, we're managing to keep things well separated and in a way that will make it easy to plug the easy auth (or any other STS) support.